benchmarks/dynamo/huggingface.py

```diff
@@ -89,42 +89,68 @@ def pip_install(package):
 
 
 SKIP = {
-    # Difficult to run and compare
-    "Reformer",
     # Fails deepcopy
-    "BlenderbotForCausalLM",
     "BlenderbotForConditionalGeneration",
-    "GPTJForCausalLM",
-    "GPTJForQuestionAnswering",
     "GPTNeoForCausalLM",
     "GPTNeoForSequenceClassification",
     # Fails with even batch size = 1
-    "DebertaV2ForMaskedLM",
-    "DebertaV2ForQuestionAnswering",
+    "GPTJForCausalLM",
+    "GPTJForQuestionAnswering",
 }
 
 # TODO - Fails even after fake tensors
 BATCH_SIZE_DIVISORS = {
     "AlbertForMaskedLM": 2,
     "AlbertForQuestionAnswering": 2,
     "AllenaiLongformerBase": 2,
+    "BartForCausalLM": 2,
     "BartForConditionalGeneration": 2,
     "BertForMaskedLM": 2,
-    "BlenderbotSmallForCausalLM": 2,
+    "BertForQuestionAnswering": 2,
+    "BlenderbotForCausalLM": 8,
+    # "BlenderbotForConditionalGeneration" : 16,
+    "BlenderbotSmallForCausalLM": 4,
     "BlenderbotSmallForConditionalGeneration": 2,
+    "CamemBert": 2,
+    "DebertaForMaskedLM": 8,
+    "DebertaForQuestionAnswering": 4,
+    "DebertaV2ForMaskedLM": 8,
+    "DebertaV2ForQuestionAnswering": 4,
+    "DistilBertForMaskedLM": 2,
+    "DistilBertForQuestionAnswering": 2,
+    "DistillGPT2": 2,
     "ElectraForCausalLM": 2,
     "ElectraForQuestionAnswering": 2,
     "GPT2ForSequenceClassification": 2,
+    # "GPTJForCausalLM" : 2,
+    # "GPTJForQuestionAnswering" : 2,
+    # "GPTNeoForCausalLM" : 2,
+    # "GPTNeoForSequenceClassification" : 2,
+    "GoogleFnet": 2,
     "LayoutLMForMaskedLM": 2,
     "LayoutLMForSequenceClassification": 2,
+    "M2M100ForConditionalGeneration": 4,
+    "MBartForCausalLM": 2,
+    "MBartForConditionalGeneration": 2,
+    "MT5ForConditionalGeneration": 2,
+    "MegatronBertForCausalLM": 4,
```

Comment:
That's weird, it makes bs=4 (16/4), but I ran it with bs=12 even with cudagraphs.

Reply:
It could be because I changed the seq_length

Reply:
I tried with a longer seq length :-)

Reply:
Discussed offline - I was trying float32, while Natalia was trying AMP. For now, we are going to stick with float32 and revisit if the results still look bad.